packages/react-api/src/api/model.js

```diff
@@ -98,14 +105,24 @@ export function executeModel(props) {
         }
       : undefined;
 
-    queryParams.spatialFilters = JSON.stringify(spatialFilters);
-    queryParams.spatialDataType = spatialDataType;
+    if (spatialFilters) queryParams.spatialFilters = JSON.stringify(spatialFilters);
+    if (spatialDataType) queryParams.spatialDataType = spatialDataType;
+    if (spatialDataColumn) queryParams.spatialDataColumn = spatialDataColumn;
```

Comment:
This only specifies which is the `spatialDataColumn` and shouldn't affect other Widgets, but it's used from the Backend perspective for determining which column they should "transform", for more info see: https://github.com/CartoDB/cloud-native/pull/17146